### ISSUE TITLE
Inconsistent z-index when rendering many circles

### ISSUE THREAD

**Issue by lukehorvat** [angular-ui/angular-google-maps#1220]
When rendering many circles on top of a transparent polygon, the circles do not have a consistent z-index. Some are rendered behind the polygon, some in front. As a result, their coloring looks different.

Plunker: http://plnkr.co/edit/FxDIJG5aORUtPUCdUDlw?p=preview

(At least I *think* it is a z-index issue, I haven't investigated it very closely yet.)

**Comment by nmccready** [comment]
Read the documentation there is a zIndex property

http://plnkr.co/edit/u0KINZ4NTLyWqC0IeYwS?p=preview

```
    <ui-gmap-polygon 
      path="polygon" 
      fill="{ opacity: 0.5 }"
      zIndex="-1">
    </ui-gmap-polygon>
  
```

It does appear that there may be an issue with circles where the zIndex is not affected, but it works for polygon and solves your issue.

**Comment by nmccready** [comment]
Fixed c083790

**Comment by lukehorvat** [comment]
Thanks!
